modules/test/playwright/tests/export-import-web/main/instance.import.spec.ts

```diff
@@ -658,101 +505,357 @@ test(
-			await apiHelpers.headlessAdminUser.getRoles('Administrator');
+		const exportFilePath = await companyExportImportPage.export(
+			`Tests 1 Items`,
+			true
+		);
 
-		await apiHelpers.headlessAdminUser.postRoleUserAccountAssociation(
-			roles.items[0].id,
-			Number(user.id)
+		await apiHelpers.delete(
+			`${apiHelpers.baseUrl}${applicationName}/${objectEntry.id}`
 		);
 
-		roles = await apiHelpers.headlessAdminUser.getRoles('Power User');
+		expect(
+			await apiHelpers.objectEntry.getObjectEntryByExternalReferenceCode(
+				applicationName,
+				objectEntry.externalReferenceCode
+			)
+		).toEqual({status: 'NOT_FOUND'});
 
-		await apiHelpers.headlessAdminUser.postRoleUserAccountAssociation(
-			roles.items[0].id,
-			Number(user.id)
+		await companyExportImportPage.import(exportFilePath, true);
+
+		const importedObjectEntry = await apiHelpers.get(
+			`${apiHelpers.baseUrl}${applicationName}/by-external-reference-code/${objectEntry.externalReferenceCode}`
 		);
 
-		await performLogout(page);
+		// The hrefs in the actions contain the object entry ID, so we need
+		// to replace it before comparing the objects
 
-		await performLogin(page, user.alternateName);
+		for (const action in importedObjectEntry.actions) {
+			if (importedObjectEntry.actions[action].href) {
+				importedObjectEntry.actions[action].href =
+					importedObjectEntry.actions[action].href.replace(
+						new RegExp(String(importedObjectEntry.id), 'g'),
+						String(objectEntry.id)
+					);
+			}
+		}
 
-		await applicationsMenuPage.goToObjects();
-		await viewObjectDefinitionsPage.clickEditObjectDefinitionLink(
-			objectDefinition.name
-		);
-		await page.getByLabel('Panel Link', {exact: true}).click();
-		await page.getByRole('option', {name: 'Object'}).click();
-		await page.getByRole('button', {name: 'Save'}).click();
-		await page.waitForTimeout(2000);
-		await applicationsMenuPage.goToObjectDefinition(objectDefinition.name);
-		await page.locator('[data-testid="fdsCreationActionButton"]').click();
-		await page.getByLabel('textField').fill('testText');
-		await page.getByRole('button', {name: 'Save'}).click();
-		await waitForAlert(
-			page,
-			'Success:Your request completed successfully.'
-		);
+		// Exclude properties that should be different
 
-		await applicationsMenuPage.goToObjectDefinition(objectDefinition.name);
+		delete importedObjectEntry.dateCreated;
+		delete importedObjectEntry.dateModified;
+		delete importedObjectEntry.id;
 
-		const objectEntryId = await page
-			.locator('table tr:first-child td:first-child')
-			.innerText();
+		delete objectEntry.dateCreated;
+		delete objectEntry.dateModified;
+		delete objectEntry.id;
 
-		const exportFilePath = await companyExportImportPage.export(
-			`${objectDefinition.name} 2 Items`
+		// Exclude friendlyUrl properties until LPD-66545 is resolved
+
+		delete objectEntry.friendlyUrlPath;
+		delete objectEntry.friendlyUrlPath_i18n;
+
+		expect(importedObjectEntry).toEqual(objectEntry);
```

Comment:
Could we use something like this instead?
```
expect(importedObjectEntry).toEqual(
			expect.objectContaining({
				...objectEntry,
				dateCreated: expect.any(String),
				dateModified: expect.any(String),
				id: expect.any(Number),
			})
		);
```

We have examples of that in in places like [export.spec.ts](https://github.com/liferay/liferay-portal/blob/291caedd4780ebd18dba11aa102a9061bf8c9823/modules/test/playwright/tests/batch-planner/main/export.spec.ts#L102) or [instance.import.spec](https://github.com/liferay/liferay-portal/blob/291caedd4780ebd18dba11aa102a9061bf8c9823/modules/test/playwright/tests/export-import-web/main/instance.import.spec.ts#L259)